Tests/ValidationTests.swift

```diff
@@ -360,10 +358,9 @@ class MultipleValidationTestCase: BaseTestCase {
         // Then
         XCTAssertNotNil(error)
 
-        if let error = error {
-            XCTAssertEqual(error.domain, ErrorDomain)
-            XCTAssertEqual(error.code, ErrorCode.statusCodeValidationFailed.rawValue)
-            XCTAssertEqual(error.userInfo[ErrorUserInfoKeys.StatusCode] as? Int, 200)
+        if let error = error as? AFError, let statusCode = error.responseCode {
+            XCTAssertTrue(error.isUnacceptableStatusCode)
+            XCTAssertEqual(statusCode, 200)
         } else {
             XCTFail("error should not be nil")
         }
```

Comment:
We should either make the fail more descriptive, or check the status like this:

```swift
XCTAssertEqual(error.responseCode, 200)
```

I'd recommend using the check I just put above.